src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationHandler.cs

```diff
@@ -159,14 +160,44 @@ protected override AuthenticationTicket AuthenticateCore()
             return new TokenResponse(oauth2Token);
         }
 
-        protected virtual async Task<AuthenticationTicket> GetUserInformationAsync(AuthenticationProperties properties, TokenResponse tokens)
+        protected virtual async Task<AuthenticationTicket> CreateTicketAsync(AuthenticationProperties properties, TokenResponse tokens)
         {
-            var context = new OAuthGetUserInformationContext(Context, Options, Backchannel, tokens)
+            var identity = new ClaimsIdentity(Options.ClaimsIssuer);
```

Comment:
I think @Tratcher mostly was arguing consistency since I changed everything to use ClaimsIssuer.  I recall that was something I questioned myself when doing it, so now is a good time to revisit to make sure we pick the most appropriate one to use...